tests/atomistics/structure/test_atoms.py

```diff
@@ -38,6 +38,7 @@ def setUpClass(cls):
         C = Atom("C").element
         cls.C3 = Atoms([C, C, C], positions=[[0, 0, 0], [0, 0, 2], [0, 2, 0]])
         cls.C2 = Atoms(2 * [Atom("C")])
+        cls.g = StructureFactory()
```

Comment:
Can we use more readable variables? In particular in tests that would be useful.

Reply:
Yes, sure. This used to be at least a little more sensible when this thing was a "Generator", but I'll make it more verbose.